src/lib/error-handler.js

```diff
@@ -2,13 +2,25 @@
 
 const { HTTPError } = require('ky-universal')
 const log = require('debug')('ipfs-http-client:lib:error-handler')
+const { isNode } = require('ipfs-utils/src/env')
 
 function isJsonResponse (res) {
   return (res.headers.get('Content-Type') || '').startsWith('application/json')
 }
 
 module.exports = async function errorHandler (input, options, response) {
-  if (response.ok) return
+  if (response.ok) {
+    // FIXME: remove when fixed https://github.com/sindresorhus/ky-universal/issues/8
+    //
+    // ky clones the response for each handler. In Node.js the response body is
+    // piped to 2 PassThroughs, one becomes the real body and the other is used
+    // in the clone.
+    //
+    // If the body in the clone is not consumed or destroyed the highwater mark
+    // will be reached (for large payloads) and stop the real body from flowing.
+    if (isNode) response.body.destroy()
```

Comment:
This...sucks...

Reply:
`ky` has some, uh, interesting caveats.

Reply:
See also: https://github.com/ipfs/js-ipfs/pull/2304/files#r322675702